### PR TITLE
Add devcontainer

### PR DESCRIPTION
**Pull request by bherbruck**

### CODE REVIEW

File: applications/web/package.json
Comment (bherbruck) [inline applications/web/package.json:1]: this was required inside the devcontainer

**Comment by av8ta** [comment]
Thanks @bherbruck! 

Just a couple minor things:

1. Could you add .pnpm-store to .gitignore in the repo root?
2. Get playwright working?

image

**Comment by av8ta** [comment]
Looks good! lgtm :) it's ready to go @bherbruck ?

**Comment by bherbruck** [comment]
I'm fine with it, confirmed playwright is working

**Pull request merged**